.eslintrc.js

```diff
@@ -1,23 +1,23 @@
 module.exports = {
-  settings: {
-    react: {
-      version: 'detect',
+    settings: {
+      react: {
+        version: 'detect',
+      },
     },
-  },
-  env: {
-    browser: true,
-    node: true,
-    es6: true,
-  },
-  plugins: ['react'],
-  extends: ['eslint:recommended', 'nextjs'],
-  parser: 'babel-eslint',
-  rules: {
-    'react/prop-types': 0,
-    'no-await-in-loop': 0,
-    'no-continue': 0,
-    'no-restricted-syntax': 0,
-    radix: 0,
-    'no-param-reassign': 0,
-  },
-};
+    env: {
+      browser: true,
+      node: true,
+      es6: true,
+    },
+    plugins: ['react'],
+    extends: ['eslint:recommended', 'plugin:react/recommended'],
```

Comment:
i would change this back to using the nextjs plugin instead. the react recommended eslint properties don't really work with nextjs that well and you'll never pass linting.